reactioncommerce/reaction
# feat: Update to node v14.11.0

- Hoping this will address #6254


Resolves #6254 
Impact: **minor**
Type: **feature|bugfix**

## Issue

Update to current stable node for general maintenance but also hopefully to avoid #6205 

## Solution

- built new docker-base images
- Build off those images here for  prod and dev

## Breaking changes

None expected.

## Testing

1. Start the api in dev mode
1. Make sure npm installs OK and the service starts and serves graphql

----

1. Start the api in prod mode
1. Ensure it starts OK and serves graphql
